charts/renku/values.yaml

```diff
@@ -510,25 +510,31 @@ ui:
     # ref: 0.1.0
 # ## Configuration for the notebooks service
 notebooks:
-  ## serverOptions allow for modifications to the notebook pod resource requests
+  ## serverOptions allow for modifications to the notebook pod resource requests and the UI rendering
   # serverOptions:
-  #   resources:
-  #     cpu_request:
-  #       displayName: Number of CPUs
-  #       type: integer
-  #       default: 1
-  #       range: [1, 4]
-  #     mem_request:
```

Comment:
Is amount of memory not needed anymore?

Reply:
Here I added just a few examples to show how the parameters are provided, not sure if I should copy the whole configuration from `notebooks` where the values are provided (non of these parameters are strictly needed for renku to work properly).

Reply:
hmm but what is the harm in leaving the `mem_request` in there? To figure out how to change it, someone deploying the platform needs to look inside the notebooks service and they might not even be aware that it is possible to modify it.